src/components/SingleDatePicker.jsx

```diff
@@ -344,7 +620,8 @@ export default class SingleDatePicker extends React.Component {
   }
 
   isHovered(day) {
-    return isSameDay(day, this.state.hoverDate);
+    const { hoverDate } = this.state || {};
```

Comment:
can you clarify that a little bit?

do you mean something like `isHovered(day, hoverDate)`? I feel like that might be odd given that it'd suddenly be different from every other modifiers method